index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./styles/style.css">
+  <title>Tribute - Albert Einstein</title>
+</head>
+<body>
+  <main id="main">
```

Comment:
I think you should separate the title which is the `header` from the `main` so that other collaborators will be able to distinguish them. The `header ` semantics is used to wrap the introductory content while the `main` contains the most unique content of the webpage.